### PR TITLE
Test & Solution for #34

### PR DESCRIPTION
**Pull request by seemcat**

### CODE REVIEW

File: solutions/34.js [hunk 1]
```diff
@@ -9,7 +9,26 @@ const solution = (arr, k) => {
...
    }
    return total;
- };
- module.exports = {
-   solution
+};
+
+// Maricris Bonzo
+
+const solution1 = (array, num) => {
```
Comment (dsope05) [inline solutions/34.js]: remove solution1 from 34.js and just include 34a.js

File: test/34.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 const expect = require('chai').expect;
 const solution = require('../solutions/34.js').solution;
+const solution1 = require('../solutions/34.js').solution;
```
Comment (dsope05) [inline test/34.js]: require from 34a.js

File: solutions/34.js (hunk 1 above)
Comment (seemcat) [inline solutions/34.js]: Resolved

File: test/34.js (hunk 2 above)
Comment (seemcat) [inline test/34.js]: Resolved

Comment (seemcat) [inline test/34.js]: How do I merge this in?

**Pull request merged**